<?xml version='1.1' encoding='UTF-8'?>
<Tag plugin="workflow-support@3.8">
  <node class="cps.n.StepAtomNode" plugin="workflow-cps@2640.v00e79c8113de">
    <parentIds>
      <string>15</string>
    </parentIds>
    <id>16</id>
    <descriptorId>org.jenkinsci.plugins.workflow.steps.durable_task.ShellStep</descriptorId>
  </node>
  <actions>
    <cps.a.ArgumentsActionImpl plugin="workflow-cps@2640.v00e79c8113de">
      <arguments>
        <entry>
          <string>script</string>
          <string>
              ./jenkins/build/maven.sh mvn -B -DskipTests clean package
              ./jenkins/build/build.sh
               </string>
        </entry>
      </arguments>
      <sensitiveVariables>
        <string>PASSWORD</string>
      </sensitiveVariables>
      <isUnmodifiedBySanitization>true</isUnmodifiedBySanitization>
    </cps.a.ArgumentsActionImpl>
    <wf.a.TimingAction plugin="workflow-api@1105.v3de5e2efac97">
      <startTime>1641182285492</startTime>
    </wf.a.TimingAction>
    <s.a.LogStorageAction/>
    <wf.a.ErrorAction plugin="workflow-api@1105.v3de5e2efac97">
      <error class="org.jenkinsci.plugins.workflow.steps.FlowInterruptedException" plugin="workflow-step-api@613.v375732a042b1">
        <stackTrace>
          <trace>org.jenkinsci.plugins.workflow.cps.CpsFlowExecution.interrupt(CpsFlowExecution.java:1146)</trace>
          <trace>org.jenkinsci.plugins.workflow.job.WorkflowRun$2.lambda$interrupt$0(WorkflowRun.java:396)</trace>
          <trace>jenkins.security.ImpersonatingScheduledExecutorService$1.run(ImpersonatingScheduledExecutorService.java:67)</trace>
          <trace>java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)</trace>
          <trace>java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)</trace>
          <trace>java.base/java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:304)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)</trace>
          <trace>java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)</trace>
          <trace>java.base/java.lang.Thread.run(Thread.java:829)</trace>
        </stackTrace>
        <suppressedExceptions class="empty-list"/>
        <result>
          <name>ABORTED</name>
          <ordinal>4</ordinal>
          <color>ABORTED</color>
          <completeBuild>false</completeBuild>
        </result>
        <causes class="java.util.Arrays$ArrayList">
          <a class="jenkins.model.CauseOfInterruption-array">
            <jenkins.model.CauseOfInterruption_-UserInterruption>
              <user>jenkins</user>
            </jenkins.model.CauseOfInterruption_-UserInterruption>
          </a>
        </causes>
        <actualInterruption>true</actualInterruption>
      </error>
    </wf.a.ErrorAction>
  </actions>
</Tag>